src/components/tooltip/tooltip.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import { Meta } from '@storybook/react'
+import { ToolTip } from './tooltip'
+import { Button } from '../button'
+
+const meta: Meta<typeof ToolTip> = {
+  title: 'Components/ToolTip',
+  component: ToolTip,
+  argTypes: {
+    tip: {
+      control: 'text',
+      description: 'Defines the tip of the tooltip.',
+    },
+    label: {
+      control: 'text',
+      description: 'Defines the label of the tooltip',
+    },
+    children: {
+      table: {
+        disable: true,
+      },
+    },
+  },
+  // This is to style the story, as the story parent container has overflow hidden.
+  decorators: [
+    (Story) => (
+      <div
+        style={{
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+          height: '20vh',
+        }}
+      >
+        <Story />
+      </div>
+    ),
+  ],
+}
+
+export default meta
+
```

Comment:
**suggestion:** I think it would be valuable to show an additional story or two for the `ElTooltip` atom directly. That way it's easy to see the visual appearance of the atom without first needing to hover or focus some other element.